middleware/configuration/source/model/transform.cpp

```diff
@@ -870,6 +872,9 @@ namespace casual
                      result.alias = null_if_empty( value.alias);
                      result.queuebase = null_if_empty( value.queuebase);
 
+                     if( value.capacity)
+                        result.capacity = common::quantity::bytes::to::string( value.capacity);
+
```

Comment:
I don't really like relying on implicit conversions to `bool` for _integers_. Should it be an `std::optional` instead? Then we should get that one could set a limit of `0B` and nothing could be enqueued to it. Might be a feature.